pkg/object/objects_test.go

```diff
@@ -816,13 +813,21 @@ spec:
 
 	for _, tt := range manifestDiffWithSelectAndIgnoreTests {
 		t.Run(tt.desc, func(t *testing.T) {
-			got, err := ManifestDiffWithSelectAndIgnore(tt.yamlStringA, tt.yamlStringB, tt.selectResources, tt.ignoreResources)
-			if err != nil {
-				t.Fatalf("unexpected error: %v", err)
-			}
-			if !strings.Contains(got, tt.want) {
-				t.Errorf("%s:\ngot:\n%v\ndoes't contains\nwant:\n%v", tt.desc, got, tt.want)
-			}
+			got, err := ManifestDiffWithSelectAndIgnore(tt.yamlStringA, tt.yamlStringB,
```

Comment:
It's generally not good Go practice to pass testing into helper functions. You could restore the original check and save some duplication by adding an additional loop for v := range []bool{true, false}

Reply:
sure, will do.